operator/cluster/upgrade.go

```diff
@@ -334,6 +334,15 @@ func createImageNamePatch(cluster crv1.Pgcluster, ccpImagePrefix, ccpImageTag st
 		Name:  "database",
 		Image: ccpImagePrefix + "/" + cluster.Spec.CCPImage + ":" + ccpImageTag,
 	}
+
```

Comment:
Wondering if we should add a note in our docs or OLM letting OLM users letting them know that the image tag they specify via `pgo upgrade` (which requires a new tag to be specified) will be overridden by whatever is specified in the new `RELATED_IMAGE_` env vars defined in the Operator deployment.